plugins/core/jetbrains-community/src/icons/AwsIcons.kt

```diff
@@ -123,6 +125,10 @@ object AwsIcons {
 
             @JvmField val SEVERITY_CRITICAL = load("/icons/resources/codewhisperer/severity-critical.svg")
         }
+
+        object InlineChat {
+            @JvmField val AWS_Q_INLINECHAT_SHORTCUT = load("/icons/resources/inlinechat/amazonq_inline_chat_shortcut.svg")
+        }
```

Comment:
hardcoded svg does not respect user configuration, or differences between OS

Reply:
ack we only show this if it's on MAC and the user configuration matches, otherwise we show text